### PR TITLE
Fixed consumer -> customer

### PR DESCRIPTION
**Pull request by NachoSoto**
Follow up to #1007.

Didn't realize I've been reading this wrong in my head the whole time, until I saw this:
Screen Shot 2021-11-30 at 12 01 28

### CODE REVIEW

File: BackendIntegrationTests/BackendIntegrationTests.swift [hunk 1]
```diff
@@ -13,11 +13,11 @@ import StoreKitTest
 
 class TestPurchaseDelegate: NSObject, PurchasesDelegate {
     var maybeCustomerInfo: CustomerInfo?
-    var consumerInfoUpdateCount = 0
+    var costumerInfoUpdateCount = 0
```
Comment (taquitos) [inline BackendIntegrationTests/BackendIntegrationTests.swift:16]: s/costumer/customer

Comment (NachoSoto) [reply BackendIntegrationTests/BackendIntegrationTests.swift:16]: 🤦

Comment (NachoSoto) [reply BackendIntegrationTests/BackendIntegrationTests.swift:16]: In my defense this isn't my native language to begin with lol

Comment (NachoSoto) [reply BackendIntegrationTests/BackendIntegrationTests.swift]: At least I did get the commit message right lol

**Review by taquitos — APPROVED** [review]
🏪 🐐🐐🐐🐐

**Pull request merged**